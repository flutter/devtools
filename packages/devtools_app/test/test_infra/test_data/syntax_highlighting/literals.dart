// Copyright 2022 The Flutter Authors
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file or at https://developers.google.com/open-source/licenses/bsd.

const a = 'aaaaa';

const list1 = [];
const list2 = <String>[];
const list3 = ['', '$a'];
const list4 = <String>['', '$a'];

const set1 = {};
const set2 = <String>{};
const set3 = {'', '$a'};
const set4 = <String>{'', '$a'};

const map1 = <String, String>{};
const map2 = {'': '$a'};
const map3 = <String, String>{'': '$a'};
