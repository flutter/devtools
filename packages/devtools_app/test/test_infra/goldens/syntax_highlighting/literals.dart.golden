>// Copyright 2022 The Chromium Authors. All rights reserved.
#^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ comment.line.double-slash.dart
>// Use of this source code is governed by a BSD-style license that can be
#^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ comment.line.double-slash.dart
>// found in the LICENSE file.
#^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ comment.line.double-slash.dart
>
>const a = 'aaaaa';
#^^^^^ storage.modifier.dart
#        ^ keyword.operator.assignment.dart
#          ^^^^^^^ string.interpolated.single.dart
#                 ^ punctuation.terminator.dart
>
>const list1 = [];
#^^^^^ storage.modifier.dart
#            ^ keyword.operator.assignment.dart
#                ^ punctuation.terminator.dart
>const list2 = <String>[];
#^^^^^ storage.modifier.dart
#            ^ keyword.operator.assignment.dart
#              ^ keyword.operator.comparison.dart
#               ^^^^^^ support.class.dart
#                     ^ keyword.operator.comparison.dart
#                        ^ punctuation.terminator.dart
>const list3 = ['', '$a'];
#^^^^^ storage.modifier.dart
#            ^ keyword.operator.assignment.dart
#               ^^ string.interpolated.single.dart
#                 ^ punctuation.comma.dart
#                   ^ string.interpolated.single.dart
#                    ^ string.interpolated.single.dart string.interpolated.expression.dart
#                     ^ string.interpolated.single.dart string.interpolated.expression.dart variable.parameter.dart
#                      ^ string.interpolated.single.dart
#                        ^ punctuation.terminator.dart
>const list4 = <String>['', '$a'];
#^^^^^ storage.modifier.dart
#            ^ keyword.operator.assignment.dart
#              ^ keyword.operator.comparison.dart
#               ^^^^^^ support.class.dart
#                     ^ keyword.operator.comparison.dart
#                       ^^ string.interpolated.single.dart
#                         ^ punctuation.comma.dart
#                           ^ string.interpolated.single.dart
#                            ^ string.interpolated.single.dart string.interpolated.expression.dart
#                             ^ string.interpolated.single.dart string.interpolated.expression.dart variable.parameter.dart
#                              ^ string.interpolated.single.dart
#                                ^ punctuation.terminator.dart
>
>const set1 = {};
#^^^^^ storage.modifier.dart
#           ^ keyword.operator.assignment.dart
#               ^ punctuation.terminator.dart
>const set2 = <String>{};
#^^^^^ storage.modifier.dart
#           ^ keyword.operator.assignment.dart
#             ^ keyword.operator.comparison.dart
#              ^^^^^^ support.class.dart
#                    ^ keyword.operator.comparison.dart
#                       ^ punctuation.terminator.dart
>const set3 = {'', '$a'};
#^^^^^ storage.modifier.dart
#           ^ keyword.operator.assignment.dart
#              ^^ string.interpolated.single.dart
#                ^ punctuation.comma.dart
#                  ^ string.interpolated.single.dart
#                   ^ string.interpolated.single.dart string.interpolated.expression.dart
#                    ^ string.interpolated.single.dart string.interpolated.expression.dart variable.parameter.dart
#                     ^ string.interpolated.single.dart
#                       ^ punctuation.terminator.dart
>const set4 = <String>{'', '$a'};
#^^^^^ storage.modifier.dart
#           ^ keyword.operator.assignment.dart
#             ^ keyword.operator.comparison.dart
#              ^^^^^^ support.class.dart
#                    ^ keyword.operator.comparison.dart
#                      ^^ string.interpolated.single.dart
#                        ^ punctuation.comma.dart
#                          ^ string.interpolated.single.dart
#                           ^ string.interpolated.single.dart string.interpolated.expression.dart
#                            ^ string.interpolated.single.dart string.interpolated.expression.dart variable.parameter.dart
#                             ^ string.interpolated.single.dart
#                               ^ punctuation.terminator.dart
>
>const map1 = <String, String>{};
#^^^^^ storage.modifier.dart
#           ^ keyword.operator.assignment.dart
#             ^ keyword.operator.comparison.dart
#              ^^^^^^ support.class.dart
#                    ^ punctuation.comma.dart
#                      ^^^^^^ support.class.dart
#                            ^ keyword.operator.comparison.dart
#                               ^ punctuation.terminator.dart
>const map2 = {'': '$a'};
#^^^^^ storage.modifier.dart
#           ^ keyword.operator.assignment.dart
#              ^^ string.interpolated.single.dart
#                ^ keyword.operator.ternary.dart
#                  ^ string.interpolated.single.dart
#                   ^ string.interpolated.single.dart string.interpolated.expression.dart
#                    ^ string.interpolated.single.dart string.interpolated.expression.dart variable.parameter.dart
#                     ^ string.interpolated.single.dart
#                       ^ punctuation.terminator.dart
>const map3 = <String, String>{'': '$a'};
#^^^^^ storage.modifier.dart
#           ^ keyword.operator.assignment.dart
#             ^ keyword.operator.comparison.dart
#              ^^^^^^ support.class.dart
#                    ^ punctuation.comma.dart
#                      ^^^^^^ support.class.dart
#                            ^ keyword.operator.comparison.dart
#                              ^^ string.interpolated.single.dart
#                                ^ keyword.operator.ternary.dart
#                                  ^ string.interpolated.single.dart
#                                   ^ string.interpolated.single.dart string.interpolated.expression.dart
#                                    ^ string.interpolated.single.dart string.interpolated.expression.dart variable.parameter.dart
#                                     ^ string.interpolated.single.dart
#                                       ^ punctuation.terminator.dart
